WalletWasabi/Services/HostedServices.cs

```diff
@@ -13,7 +13,7 @@ public class HostedServices : IDisposable
 
 	private List<HostedService> Services { get; } = new List<HostedService>();
```

Comment:
```suggestion
	private List<HostedService> Services { get; } = new();
```

Reply:
For fields and variables there are none left, but for properties turning that on doesn't detect those.